backend/chainlit/step.py

```diff
@@ -140,7 +140,7 @@ def sync_wrapper(*args, **kwargs):
                     try:
                         if result and not step.output:
                             step.output = result
-                    except:
+                    except Exception as e:
                         step.is_error = True
                         step.output = str(e)
```

Comment:
Wow, was this a broken exception handler here?

Reply:
Indeed, this was introduced in [4e0baab](https://github.com/Chainlit/chainlit/commit/4e0baabe3fd48abdd2f9998395b02c40e1f3d9e4#diff-4677e8359efad4a24ef6e70a44fd94c6549e46d017c3a0a9692bfd7d890bb75eR115).